src/nomad_material_processing/general/schema.py

```diff
@@ -0,0 +1,278 @@
+        ),
+        unit='second',
+    )
+    annealing_reagents = SubSection(
+        section_def=CompositeSystemReference,
+    )
+
+
+class AnnealingRecipe(Annealing, Recipe, EntryData):
+    """
+    A Recipe for an annealing process.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['datetime', 'samples']},
+    )
+
+
+class Cleaning(Process, EntryData):
+    """
+    Class autogenerated from yaml schema.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['steps']},
+    )
+    recipe = Quantity(
+        type=Recipe,
+        description=""" The recipe used for the process. If a recipe is found,
+           all the data is copied from the Recipe within the Process.
+           """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.ReferenceEditQuantity,
+        ),
+    )
+    temperature = Quantity(
+        type=np.float64,
+        description='The temperature of the annealing process.',
+        a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'celsius'},
+        unit='celsius',
+    )
+    duration = Quantity(
+        type=np.float64,
+        description='The elapsed time since the annealing process started.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity, defaultDisplayUnit='minute'
+        ),
+        unit='second',
+    )
+    cleaning_reagents = SubSection(
+        section_def=CompositeSystemReference,
+    )
+
+
+class CleaningRecipe(Cleaning, Recipe, EntryData):
+    """
+    A Recipe for an cleaning process.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['datetime', 'samples']},
+    )
+
+
+class BackSideCoating(Process, EntryData):
+    """
+    Class autogenerated from yaml schema.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['steps']},
+    )
+    recipe = Quantity(
+        type=Recipe,
+        description=""" The recipe used for the process. If a recipe is found,
+           all the data is copied from the Recipe within the Process.
+           """,
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.ReferenceEditQuantity,
+        ),
+    )
+    temperature = Quantity(
+        type=np.float64,
+        description='The temperature of the annealing process.',
+        a_eln={'component': 'NumberEditQuantity', 'defaultDisplayUnit': 'celsius'},
+        unit='celsius',
+    )
+    duration = Quantity(
+        type=np.float64,
+        description='The elapsed time since the annealing process started.',
+        a_eln=ELNAnnotation(
+            component=ELNComponentEnum.NumberEditQuantity, defaultDisplayUnit='minute'
+        ),
+        unit='second',
+    )
+    coating_reagents = SubSection(
+        section_def=CompositeSystemReference,
+    )
+
+
+class BackSideCoatingRecipe(BackSideCoating, Recipe, EntryData):
+    """
+    A Recipe for a back side coating process.
+    """
+
+    m_def = Section(
+        a_eln={'hide': ['datetime', 'samples']},
+    )
```

Comment:
This is just a special case of a coating process. I would not add that to general schemas.